src/Evolution/Ringdown/CMakeLists.txt

```diff
@@ -0,0 +1,38 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+set(LIBRARY Ringdown)
+
+add_spectre_library(${LIBRARY})
+
+spectre_target_sources(
+  ${LIBRARY}
+  PRIVATE
+  StrahlkorperCoefsInRingdownDistortedFrame.cpp
+  )
+
+spectre_target_headers(
+  ${LIBRARY}
+  INCLUDE_DIRECTORY ${CMAKE_SOURCE_DIR}/src
+  StrahlkorperCoefsInRingdownDistortedFrame.hpp
+  )
+
+target_link_libraries(
+  ${LIBRARY}
+  PUBLIC
```

Comment:
I think you can remove
- `Boost::boost`
- `DomainStructure`
- `ErrorHandling`
- `Options`
- `ParallelInterpolation`
- `Serialization`